handler/interceptors.go

```diff
@@ -137,6 +155,10 @@ func interceptMonitoring(srv interface{}, ss grpc.ServerStream, info *grpc.Strea
 }
 
 func (interceptor *rateLimitInterceptor) intercept(srv interface{}, ss grpc.ServerStream, info *grpc.StreamServerInfo, handler grpc.StreamHandler) error {
+
+	if (interceptor.processRequest == STOP_PROCESING_ANY_REQUEST) {
+		return fmt.Errorf("No requests are currently being processed, please try again later")
```

Comment:
Probably gRPC error code should be returned, for example:
```suggestion
		return status.New(codes.Unavailable, "No requests are currently being processed, please try again later").Err()
```

Reply:
Not sure, `Unavailable` suits case well

Reply:
unavailable seems most appropriate